File: Ad-Hoc/__3000__/3129/3129.py
stickers = set()
repeated_stickers = 0

n_stickers = int(input())
for _ in range(n_stickers):
    sticke = int(input())
    length = len(stickers)
    stickers.add(sticke)
    current_len = len(stickers)
    if current_len == length:
        repeated_stickers += 1
        
print(len(stickers))
print(repeated_stickers)

File: Ad-Hoc/__3000__/3129/3129.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("\n").map(Number)
const n = input.shift()

let stickers = new Set()
let repeated_stickers = 0

for(let i = 0; i < n; i++){
    const sticker = input[i]
    const curr_lenght = stickers.size

    stickers.add(sticker)

    repeated_stickers += curr_lenght == stickers.size ? 1 : 0
}

console.log(stickers.size)
console.log(repeated_stickers)
